Author: Pino Toscano <pino@kde.org>
Description: preliminary GNU/Hurd support
 Adds "support" for GNU/Hurd by defining PATH_MAX where needed.
 Not forwarded, better fixes should be done upstream.
Forwarded: no
Fixed-For: 4.5.0
--- a/kholidays/parsers/plan1/parseholiday.c
+++ b/kholidays/parsers/plan1/parseholiday.c
@@ -170,6 +170,10 @@
 
 #include <limits.h>
 
+#ifndef PATH_MAX
+# define PATH_MAX 1024
+#endif
+
 /*** Macro definitions and constants ***/
 /*
  * Before you mail and complain that the following macro is incorrect,
--- a/kholidays/parsers/plan1/parseholiday.y
+++ b/kholidays/parsers/plan1/parseholiday.y
@@ -49,6 +49,10 @@
 
 #include <limits.h>
 
+#ifndef PATH_MAX
+# define PATH_MAX 1024
+#endif
+
 /*** Macro definitions and constants ***/
 /*
  * Before you mail and complain that the following macro is incorrect,
